beesdoo_purchase/views/purchase_order.xml

```diff
@@ -0,0 +1,14 @@
+<openerp>
+	<data>
+		<record model="ir.ui.view" id="beesdoo_purchase_order_form_view">
+			<field name="name">beesdoo.purchase.order.form.view</field>
+			<field name="model">purchase.order</field>
+			<field name="inherit_id" ref="purchase.purchase_order_form" />
+			<field name="arch" type="xml">
+				<field name="product_id" position="attributes">
```

Comment:
It's not correct:
                <field name="product_id" position="attributes">
                    <attribute name="domain">[('main_seller_id','=', parent.partner_id)]</attribute>
                </field>

inside it's attribute without S then name="domain" and then the parent.partner_id should not be quoted